consensus/types/src/consts.rs

```diff
@@ -0,0 +1,21 @@
+pub mod altair {
+    pub const TIMELY_HEAD_FLAG_INDEX: u32 = 0;
+    pub const TIMELY_SOURCE_FLAG_INDEX: u32 = 1;
+    pub const TIMELY_TARGET_FLAG_INDEX: u32 = 2;
+    pub const TIMELY_HEAD_WEIGHT: u64 = 12;
+    pub const TIMELY_SOURCE_WEIGHT: u64 = 12;
+    pub const TIMELY_TARGET_WEIGHT: u64 = 24;
+    pub const SYNC_REWARD_WEIGHT: u64 = 8;
+    pub const PROPOSER_WEIGHT: u64 = 8;
+    pub const WEIGHT_DENOMINATOR: u64 = 64;
+    pub const INACTIVITY_SCORE_BIAS: u64 = 4;
```

Comment:
I notice that `INACTIVITY_SCORE_BIAS` and `INACTIVITY_PENALTY_QUOTIENT_ALTAIR` are present here and in `ChainSpec`, but I'm yet to figure out why.

Reply:
ooh I don't think there's any good reason for that, we can probably consolidate